kafka/server.config

```diff
@@ -15,7 +15,7 @@ delete.topic.enable=true
 #     listeners = listener_name://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
-listeners=PLAINTEXT://192.168.0.58:9092
+listeners=PLAINTEXT://172.17.5.38:9092
```

Comment:
So the 172 address was added by accident, I now try to determine the IP at the time of running the install script, but what would be a good default in case this fails? 127.0.0.1 ?

Reply:
As a default, we could leave it empty (binds to default interface) or set it to 0.0.0.0 (binds to all interfaces).